src/components/Edit/Edit.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react"
+import { updateDynamo } from "../PhotoAnalysis/utils"
+import { LabelType } from "../../../types/label"
+
+type Props = {
+  recordData: any
+}
+
+const Edit: React.FC<Props> = ({ recordData }) => {
+  const [labelsArray, setLabelsArray] = useState<LabelType[]>([])
+  // TODO store labels in a state
+  useEffect(() => {
```

Comment:
Can you please explain the choice for a useEffect here? I'm not disagreeing, I just don't understand?